backend/requirements-test.txt

```diff
@@ -17,3 +16,15 @@ testing.postgresql>=1.3.0
 flask-shell-ipython
 python-dotenv
 watchdog
+
+# All requirements for oio_rest
+-e git+https://github.com/magenta-aps/mox@development#egg=oio-rest&subdirectory=oio_rest
+pytz==2018.5
+Jinja2==2.10
+MarkupSafe==1.0
+Werkzeug==0.14.1
+psycopg2cffi-compat==1.1
+python3-saml==1.4.1
```

Comment:
Is `python3-saml` still a dependency of `oio_rest`?

Reply:
No, it isn't. I'll clean these up.